OneCommunityGlobal/HighestGoodNetworkApp
# ruike_add_active_indicator

# Description
<img width="677" alt="Screen Shot 2023-11-18 at 11 32 21 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114443664/13634e92-820a-49a0-ad25-7a12fd2b0e2a">

## Related PRS (if any):
N/A

## Main changes explained:
In the Teams->Members
Add green/grey dot to indicate if user is active or not

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin/owner
4. go to Other Links -> Teams -> Members
5. check all users have a corresponding dot to indicate if they are active or not
6. go to Other Links -> User Management
7. click one user's green dot to switch it to inactive
8. go to Other Links -> Teams -> Members
9. check that user's dot became grey

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114443664/8bedbbe5-d9bb-4210-8441-e2388f18aca5
